ui/ui.go

```diff
@@ -144,17 +144,40 @@ func newModel(cfg Config) tea.Model {
 		cfg: cfg,
 	}
 
-	return model{
+	m := model{
 		common: &common,
 		state:  stateShowStash,
 		pager:  newPagerModel(&common),
 		stash:  newStashModel(&common),
 	}
+
+	info, _ := os.Stat(cfg.Path)
```

Comment:
Let me know what I could do on an error here given this function does not accept returning `err`.